hayeon/programmers-2/영어-끝말잇기.js

```diff
@@ -0,0 +1,28 @@
+function solution(n, words) {
+  let answer = [0, 0];
+
+  for (let i = 0; i < words.length; i++) {
+    const word = words[i];
+    const person = (i % n) + 1;
+    const turn = Math.ceil((i + 1) / n);
+
+    if (i > 0) {
+      const last = words[i - 1].split("").pop();
+
+      if (i > words.indexOf(word) || words[i][0] !== last) {
+        answer = [person, turn];
+        break;
+      }
+    }
+  }
+
+  return answer;
+}
```

Comment:
```suggestion
function solution(n, words) {
  for (let i = 0; i < words.length; i++) {
    const word = words[i];
    const person = (i % n) + 1;
    const turn = Math.ceil((i + 1) / n);

    if (i > 0) {
      const last = words[i - 1].split("").pop();
      const isValid = i <= words.indexOf(word) && words[i][0] === last;

      if (!isValid) return [person, turn];
    }
  }

  return [0, 0];
}
```
- 풀이잘봣씁니다 고생하셨씁니다.
- if문의 복잡한 코드는 따로 변수로 빼서 의미를 명확하게해서 가독성을 키워보시는 것은 어떨까요?(isValid)
- answer에 할당하는게 아닌 곧바로 return을 하면 answer라는 변수도 불필요해집니다